yaml/csi-driver/edge/nimble-csp.yaml

```diff
@@ -4,6 +4,13 @@
 
 ---
 
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: hpe-storage
+
+---
+
```

Comment:
I don't think we should create the namespace as part of the deployment as you do a `kubectl delete` it will wipe any secrets created there.

Reply:
ok, then we document this? Users will easily hit this with yaml.

Reply:
Users should follow the instructions found on SCOD and we'll document it there.

Reply:
cool